### PR TITLE
Reject unknown arguments in WindowInto.

### PR DESCRIPTION
**Pull request by robertwb**
This is important for usability in case there are typos or,
say, triggerfn is used instead of trigger.

Also updates docstring of this method.

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

### CODE REVIEW

**Comment by robertwb** [comment]
R: @youngoli

**Comment by robertwb** [comment]
The java failures are unrelated.

File: sdks/python/apache_beam/transforms/core.py [hunk 1]
```diff
@@ -1608,10 +1608,18 @@ def __init__(self, windowfn, **kwargs):
 
     Args:
       windowfn: Function to be used for windowing
+      trigger: (optional) Trigger used for windowing, or None for default.
+      accumulation_mode: Accumulation mode used for windowing,
```
Comment (youngoli) [inline sdks/python/apache_beam/transforms/core.py]: Should this also be marked optional? It seems optional looking at the code.

Comment (robertwb) [reply sdks/python/apache_beam/transforms/core.py]: It's sometimes optional. Updating the comment.

**Pull request merged**